srtp/srtp.c

```diff
@@ -2039,16 +2037,15 @@ static srtp_err_status_t srtp_unprotect_aead(srtp_ctx_t *ctx,
     /*
      * Set the AAD for AES-GCM, which is the RTP header
      */
-    aad_len = (size_t)(enc_start - (uint8_t *)hdr);
-    status =
-        srtp_cipher_set_aad(session_keys->rtp_cipher, (uint8_t *)hdr, aad_len);
+    aad_len = (size_t)(enc_start - srtp_hdr);
```

Comment:
Does this actually produce a warning? I would assume this will convert without any issue.

Reply:
No and looks like it is just left over from refactoring, good catch. I searched and there is a few more, will update pr before merging.

Reply:
There are several. Line 1827, 1898, 2008, 2040 (this one), 2283, and 2652.  That's all I saw.

Reply:
done